packages/react/src/themes/teams/components/Chat/chatItemStyles.ts

```diff
@@ -22,6 +23,9 @@ const getChatMessageEvaluatedStyles = (p: ChatItemProps) => ({
       paddingTop: pxToRem(5),
       paddingBottom: pxToRem(7),
       ...getPositionStyles(p),
+      [chatMessageContentClassNameSelector]: {
```

Comment:
I had to conditionally change the display prop on the content, because if the author and timestamp are not visible in Teams theme, the reactions should appear after the content. No better idea in the moment (tried added float elements, but then the paddings are broken...) I didn't want to create additional element for wrapping this, so this seemed like the only option.